KLR/Core/Operators.lean

```diff
@@ -1,73 +1,86 @@
 /-
 Copyright (c) 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Paul Govereau, Sean McLaughlin
+Authors: Paul Biberstein, Paul Govereau, Pavel Potapov, Sean McLaughlin, Markus De Medeiros
 -/
+import KLR.Core.Tensor
+import KLR.Serde.Attr
+import KLR.Serde.Elab
+import KLR.Util
+import Lean
 
 /-
 # Definition of operators
 
-Operators can appear in a call node, and can be thought of as functions that
-take arguments and return results. However, an operator may have several
-variations, and these variations show up as parameters within the operator and
-not as arguments of the call node. A typical example is:
+These definitions follow the hardware ISA as close as possible.
+-/
+namespace KLR.Core
+open Lean (FromJson ToJson)
+open Serde (FromCBOR ToCBOR)
+open Util (FromSexp ToSexp)
 
-  .store t [] (.call (.operator (.tensorScalar ts)) [a,b] [])
+instance : ToJson Float32 where
```

Comment:
I think these should either be marked local, or go in Common. It would be annoying to include a file and get a new class instance that was not obviously related to the types in the file I included.